spec/support/jasmine-browser.json

```diff
@@ -5,6 +5,7 @@
     "govuk_publishing_components/all_components-*.js",
     "govuk_publishing_components/modules-*.js",
     "govuk_publishing_components/analytics-*.js",
+    "govuk_publishing_components/load-analytics-*.js",
```

Comment:
This is interesting - does `load-analytics` need the `-*` on the end or were you just following the convention? Does it work without?

Reply:
Yeah, the tests fail without the `-*`, not sure why though.

Reply:
Fair enough. Some Rails magic, I guess.